scripts/patch-release/patch-release.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/blang/semver/v4"
+	"github.com/cri-o/cri-o/internal/version"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/release-sdk/git"
+	"sigs.k8s.io/release-sdk/github"
+	"sigs.k8s.io/release-utils/env"
+	"sigs.k8s.io/release-utils/util"
+)
+
+const (
+	githubTokenEnvKey = "GITHUB_TOKEN"
+	gitRemoteEnvKey   = "REMOTE"
+	orgEnvKey         = "ORG"
+	versionFile       = "internal/version/version.go"
+	branchPrefix      = "release-"
+)
+
+func main() {
+	logrus.SetFormatter(&logrus.TextFormatter{DisableTimestamp: true})
+	if err := run(); err != nil {
+		logrus.Fatalf("Unable to %v", err)
+	}
+}
+
+func run() error {
+	if !env.IsSet(githubTokenEnvKey) {
+		return fmt.Errorf(
+			"run: $%s environment variable is not set", githubTokenEnvKey,
+		)
+	}
+	if !env.IsSet(orgEnvKey) {
+		return fmt.Errorf(
+			"run: $%s environment variable is not set %s",
+			orgEnvKey,
+			"(should be set to your CRI-O fork organization, like 'gh-name')",
+		)
+	}
+	remote := env.Default(gitRemoteEnvKey, "origin")
+	logrus.Infof("Using repository fork remote: %s", remote)
+
+	org := os.Getenv(orgEnvKey)
+	logrus.Infof("Using repository fork organization: %s", org)
+
+	repo, err := git.OpenRepo(".")
+	if err != nil {
+		return fmt.Errorf("open local repo: %w", err)
+	}
+
+	currentBranch, err := repo.CurrentBranch()
+	if err != nil {
+		return fmt.Errorf("get current branch: %w", err)
+	}
+	logrus.Infof("Using current branch: %s", currentBranch)
+
+	newVersion, err := incVersion(version.Version, currentBranch)
+	if err != nil {
+		return fmt.Errorf("increment version: %w", err)
+	}
+
+	logrus.Infof("Using new version: %s", util.SemverToTagString(newVersion))
+
+	if err := updateVersionAndCreatePR(
+		repo, newVersion, currentBranch, org, remote,
+	); err != nil {
+		return fmt.Errorf("update version in local repository: %w", err)
+	}
+
+	return nil
+}
+
+func incVersion(tag, branch string) (res semver.Version, err error) {
```

Comment:
Please fix the linter here: `incVersion` - `branch` is unused

Reply:
added this change as requested